ddtrace/internal/datadog/profiling/include/exporter.hpp

```diff
@@ -7,6 +7,7 @@
 
 #include <array>
 #include <chrono>
+#include <deque>
 #include <memory>
```

Comment:
Minor: is the vector include still needed?

Reply:
Yes